jormungandr/src/rest/v0/logic.rs

```diff
@@ -658,3 +659,12 @@ pub async fn get_committees(context: &Context) -> Result<Vec<String>, Error> {
         .map(|cid| cid.to_string())
         .collect())
 }
+
+pub async fn get_active_vote_plans(context: &Context) -> Result<Vec<VotePlan>, Error> {
```

Comment:
I'm taking all of them right now. But is it correct though? Which ones do we consider the active ones?

Reply:
all the one returned by the ledger() `active_vote_plans`